topics/visual-studio/index.md

```diff
@@ -0,0 +1,18 @@
+---
+aliases: vs
+created_by: Microsoft Corporation
+display_name: Visual Studio
+logo: visual-studio.png
+related: vscode-theme, vscode-extension, text-editor
+released: 1997
+short_description: Development software suite for Windows & MacOS made to generate apps, web-services, desktop apps and mobile apps.
+topic: visual-studio
+url: https://visualstudio.microsoft.com/
+wikipedia_url: https://en.wikipedia.org/wiki/Microsoft_Visual_Studio
+---

```

Comment:
```suggestion
---
```

Reply:
![](https://i.imgur.com/G5QcToq.png)
+ I don't have any space after any of the `---`

Reply:
@HideakiAtsuyo You had a `\r`